### ISSUE TITLE
Incorrect warning: "app 'BackGround' does not have the proper Case 'Background'"

### ISSUE THREAD

**Issue by lardconcepts** [ossobv/asterisklint#15]
```app 'BackGround' does not have the proper Case 'Background'```

But...

https://wiki.asterisk.org/wiki/display/AST/Asterisk+14+Application_BackGround

**Comment by wdoekes** [comment]
I am aware of that:
https://github.com/ossobv/asterisklint/blob/8a1df5e2f110b5557d4ad506ce57d0a98d789e70/asterisklint/app/vall/builtin.py#L36

But I cannot reasonably expect people to spell it like that.

**Comment by lardconcepts** [comment]
OK, I understand - the thing is though, for people using syntax highlighter extensions and plugins for IDE and code editors like Notepad++, Atom, SublimeText etc, it "corrects" to the official syntax.
Would you consider a "lenient" option in cases like this? :)
